_pages/learning.md

```diff
@@ -33,3 +33,16 @@ Get your hands on Crystal with a series of exercises, and compare your solution
 to that of the many members who tried already.
 
 [Crystal track on Exercism](https://exercism.org/tracks/crystal/)
+
+<span id="codecrafters"></span>
```

Comment:
**suggestion:** Remove this explicit anchor. It's unnnecessary because the headline already creates it implicitly.